components/Hotel/Amenities.vue

```diff
@@ -4,17 +4,30 @@
 
     <div class="mt-5 flex flex-col flex-wrap gap-3">
       <div
-        class="flexitems-center justify-center"
-        v-for="data in amenities"
+        class="flex items-center justify-start"
+        v-for="(data, index) in visibleAmenities"
```

Comment:
Cemu sluzi ovaj index?

Reply:
the index must not be an obstacle, obrisao index i :key="index"